orderer/consensus/kafka/consenter.go

```diff
@@ -103,8 +103,8 @@ func (consenter *consenterImpl) HandleChain(support consensus.ConsenterSupport,
 	return ch, nil
 }
 
-func (c *consenterImpl) JoinChain(support consensus.ConsenterSupport, joinBlock *cb.Block) (consensus.Chain, error) {
-	return nil, errors.New("the Kafka orderer does not support JoinChain")
+func (c *consenterImpl) IsChannelMember(joinBlock *cb.Block) (bool, error) {
```

Comment:
We could instead of making `IsChannelMember` a part of the consenter interface, having a separate `ClusterConsenter` interface that consensus types can optionally implement.  It would be a simple type assertion on the other side, and would make it easier to differentiate between a 'not supported' type error, and a genuine one.

Reply:
Good idea.